toil-appliance/test/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+mkdir ./tempDir
+
+LOCAL_PATH=$(cd ./tempDir; pwd)
+LEADER_PATH=/test/data
+UUID=$(python -c 'import uuid; print uuid.uuid4()')
+LEADER_IP=172.17.0.2
```

Comment:
@hannes-ucsc this still points to mesos' default IP- changing it to any other private IP made mesos complain. I figured this was ok for now but let me know if I should try to fix this.